packages/tables/src/Columns/Concerns/CanWrapHeader.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Filament\Tables\Columns\Concerns;
+
+use Closure;
+
+trait CanWrapHeader
+{
+    protected bool | Closure $isHeaderWrapped = false;
+
+    public function wrapHeader(bool | Closure $condition = true): static
+    {
+        $this->isHeaderWrapped = $condition;
+
+        return $this;
+    }
+
+    public function isHeaderWrapped(): bool
+    {
+        return (bool) $this->evaluate($this->isHeaderWrapped);
```

Comment:
@danharrin, only in some places we cast the `evaluate` result to `bool`. Should we do this everywhere for consistency?

Reply:
Yes please

Reply:
Have added it to the roadmap, assigned to me. Will PR for v3 to avoid conflicts.